pkg/utils/pretty/pretty.go

```diff
@@ -85,3 +86,12 @@ func Taint(t v1.Taint) string {
 	}
 	return fmt.Sprintf("%s=%s:%s", t.Key, t.Value, t.Effect)
 }
+
+var matchFirstCap = regexp.MustCompile("(.)([A-Z][a-z]+)")
+var matchAllCap = regexp.MustCompile("([a-z0-9])([A-Z])")
+
+func ToSnakeCase(str string) string {
```

Comment:
Let's consider adding a test to this so that we have confidence that this logic is working